internal/api/v1/service/delete.go

```diff
@@ -125,6 +126,8 @@ func Delete(c *gin.Context) apierror.APIErrors {
 
 	boundAppNames = helpers.UniqueStrings(boundAppNames)
 
+	logger.Info("bound to", "apps", boundAppNames, "count", len(boundAppNames), "unbind?", deleteRequest.Unbind)
```

Comment:
The k/v pairs are meant to be used in structured logs. Having a key with `?` is possibly an issue with parser, greps or whatever, also in json.

```suggestion
	logger.Info("bound to", "apps", boundAppNames, "count", len(boundAppNames), "unbind", deleteRequest.Unbind)
```

Reply:
done with commit f59a6fe6a7e2948f7004856dd35ce18afc625817